Tasks/DecryptFile/decrypt.ps1

```diff
@@ -0,0 +1,57 @@
+#
+#  Copyright (c) Microsoft. All rights reserved.  
+#  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+param(
+    [string]$cipher,
+    [string]$inFile,
+    [string]$passphrase,
+    [string]$outFile,
+    [string]$cwd
+)
+
+Write-Verbose "Entering script DecryptFile.ps1"
+Write-Host "cipher = $cipher"
+Write-Host "inFile = $inFile"
+Write-Host "passphrase = $passphrase"
+Write-Host "outFile = $outFile"
+Write-Host "cwd = $cwd"
+
+# Import the Task dll that has all the cmdlets we need for Build
+import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
+import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
+
+#Verify curl is installed correctly
+$openssl = Get-Command openssl.exe
```

Comment:
GCM writes to the error pipeline but it doesn't throw. If you want to suppress the error from GCM, you can do: $openssl = Get-Command openssl.exe -ErrorAction SilentlyContinue

I think the way you have it is fine though. Just thought I'd mention it as fyi since this looked like your original intent in the previous version of the PR.